### ISSUE TITLE
🐛 Biome not detected in VS Code workspace

### ISSUE THREAD

**Issue by marksamman** [biomejs/biome#978]
### Environment information

```block
CLI:
  Version:                      1.4.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v21.2.0"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

1. Install biomejs.biome extension for VS Code
2. Open a workspace in VS Code where there's a project containing Biome installed in node_modules and a biome.json configuration file
3. Save a file under that project that can be formatted
4. Biome can't find the Biome executable in workspace projects node_modules, and even if I let the extension install Biome globally (which I rather not do because I would like it to use project pinned version) it doesn't format the file anyway, probably because it doesn't find biome.json.

### Expected result

The Biome version installed in node_modules and biome.json should be found automatically. This works fine with for example the prettier and eslint VS Code extensions. With Biome it also works fine when the project in the workspace is opened as a folder in VS Code, but not when the project is opened as part of a workspace.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Closing as duplicate of https://github.com/biomejs/biome-vscode/issues/18

If you happen to arrive here, please refer to the above issue and help us to find a good solution
